.github/workflows/ci.yml

```diff
@@ -37,19 +30,21 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ~/.local/
-          key: ${{ runner.os }}-local-v4
+          key: ${{ runner.os }}-local-v5-${{ hashFiles('.github/scripts/install-*') }}
```

Comment:
what happened here?

Reply:
I added a hash of the scripts so it gets invalidated in case we ever update a dependency. Right now the cache has a fixed name and it'll still use old binaries even if we update the scripts.

Reply:
sounds like a good change 👍